frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -446,12 +450,33 @@ function RunAiModelTab({ aIJobModalState }: { aIJobModalState: string }) {
                 </Card>
               </Radio.Button>
             </Tooltip>
+            <Tooltip title="Coming soon">
+              <Radio.Button
+                className="aIJobSelection"
+                checked={aIJobModalState === "align_sections"}
+                disabled={!Store.getState().activeUser?.isSuperUser}
```

Comment:
Here is still a access to the `isSuperUser` field of the active user, although it was already selected above. You can just use the selected `isSuperUser` variable.
```suggestion
                disabled={!isSuperUser}
```